### ISSUE TITLE
New Rendition

### ISSUE THREAD

**Issue by Alviks** [Alviks/Paulina_Graphic_design#4]
Hi Paulina, 
We had our meeting with Daniel. He's not exactly forward thinking for going for a more spectacular version of a logo for our climbing division. So we are hindered to the logo they have and we've been asked to just "Add something" to it to somehow make a distinction between painter crew and climber crew. We thought maybe over the top of the Logo we can write something like "Klatter Division" and possibly some kind of colorful vertical bar to the right of the logo with something to do with climbers - like a silhouette of a industrial climber or something w ropes or to do with rope access... Maybe not so fancy but simple and able to see that it is a climber department.. 
I added a really bad mockup of it.. im sure you have 1000 ideas better than i can come up with. Let me know what you think?

/Eddie
![klatterdiv](https://user-images.githubusercontent.com/40559863/54432023-4bf32280-4728-11e9-901a-d3b30912df3c.png)

**Comment by paulinazbylut** [comment]
[AMK Klatter.pdf](https://github.com/Alviks/Paulina_Graphic_design/files/3124245/AMK.Klatter.pdf)
Hello !
I was trying to do something with the logo, I really think there is no sense with adding any climbing accents - it actually look stupid (in my opinion ;) ) a little bit unprofessional I think. There is too mach going on with the typography and the colors... Maybe we could try to give up two pieces of this strange typography repetitions, maybe we could stay with just one superscription and than just add Klatter... It looks fine, simple and clear, what do you think?

warms
P

**Comment by Alviks** [comment]
yeah im absolutely in agreement. Is just too stupid idea to throw a climber onto this already busy logo.. we go with (KISS) - "keep it simple stupid"..... 
so question is what color to go with what clothing? maybe yellow or red on blue clothing and blue one on a white tshirt? have any opinions on that?

**Comment by paulinazbylut** [comment]
Yeah, I think white logo on blues is the best option.
